Add warning about cagg_migrate issues in 2.8.0 (#1714)
timescale/docs: api/cagg_migrate.md

For more information, see the [migration how-to guide][how-to-migrate].

<highlight type="warning">

> There are known issues with `cagg_migrate()` in 2.8.0.
> Please upgrade to at least 2.8.1 before using it.

Reviewer: ```suggestion
There are known issues with `cagg_migrate()` in version 2.8.0.
Upgrade to version 2.8.1 or above before using it.
```
